src/vuestic-theme/vuestic-components/va-checkbox/va-checkbox-docs.md

```diff
@@ -13,13 +13,19 @@
   :errorCount="3"
   label="With errors"
 />
-```  
+<va-checkbox
+  v-model="array"
+  array-value="one"
+/>
+```
 
 **Props**
 * `id` - String
 * `name` - String
 * `label` - String - label to the right of checkbox
 * `value` - Boolean - main value
+* `array` - Array<String>
+* `arrayValue` - String - if arrayValue is included in value array - checkbox should be selected
```

Comment:
Probably no type validation here would make more sense.

Definitely could be string, object, symbol, number etc.

Reply:
"Any" type can cover them all.
Is it ok?

Reply:
I think you can just skip type.
There is no `any` in JS, and we're not going TS yet. :)